templates/templates/blockified/archive-product.html

```diff
@@ -1,8 +1,41 @@
 <!-- wp:template-part {"slug":"header"} /-->
-<!-- wp:group {"layout":{"inherit":true}} -->
+
+<!-- wp:group {"layout":{"inherit":true,"type":"constrained"}} -->
 <div class="wp-block-group">
-	<!-- wp:paragraph -->
-	<p>Archive product blockified</p>
-	<!-- /wp:paragraph --></div>
+	<!-- wp:query {"query":{"perPage":9,"pages":0,"offset":0,"postType":"product","order":"asc","orderBy":"title","author":"","search":"","exclude":[],"sticky":"","inherit":true,"__woocommerceAttributes":[],"__woocommerceStockStatus":["instock","outofstock","onbackorder"]},"displayLayout":{"type":"flex","columns":3},"namespace":"woocommerce/product-query","align":"wide"} -->
+	<div class="wp-block-query alignwide">
+		<!-- wp:query-title {"type":"archive","showPrefix":false} /-->
+
+		<!-- wp:term-description /-->
```

Comment:
For the `Product Catalog` template there will be term, or term description however it is displayed in the Site Editor. Is it possible to conditionally render this if the user is viewing the templates:

* Products by Category
* Products by Attribute
*Products by Tag (which by the way is not displaying the blockified version at this point, is this correct?)

Reply:
Hmm, good catch. 

> Is it possible to conditionally render this if the user is viewing the templates:

Currently, all archive templates use the same `archive-product.html` template (see #7712). I think we can do one of:

- Create a new archive template with description, fallback all other templates to use the one with description instead of the `archive-product.html`.
- Chime in the template parsing process to remove the description block for Product Catalog template.
- Remove the description block.

@albarin @tjcafferkey  what do you think is the best direction here?

Reply:
>Chime in the template parsing process to remove the description block for Product Catalog template.

This feels a little bit hacky and we could end up setting a precedent for other people to do the same thing for other template scenarios which might not be ideal. I'm open to people disagreeing though.

>Create a new archive template with description, fallback all other templates to use the one with description instead of the archive-product.html.

This feels like a better option but comes with the overhead of maintaining two templates, would that be a problem?

Reply:
You read my mind Tom! If I have to choose one now, that'd be the second direction. But I'm not happy with either of them.

Reply:
I missed this block in [template conversion PR](https://github.com/woocommerce/woocommerce-blocks/pull/8248). I'll include it there as well for Products by Category, Products by Attribute and Products by Tag

Reply:
> > Create a new archive template with description, fallback all other templates to use the one with description instead of the archive-product.html.
> 
> This feels like a better option but comes with the overhead of maintaining two templates, would that be a problem?

Not completely sure, but I think this would have more implications for the whole fallback system in place. Currently, if you make a modification to the `Product Catalog` template you should see it on the other three (cat, tag, attr). If we have a separate template with the description I think we'd loose this 🤔 

> Currently, all archive templates use the same archive-product.html template (see https://github.com/woocommerce/woocommerce-blocks/pull/7712). I think we can do one of:
> - Create a new archive template with description, fallback all other templates to use the one with description instead of the archive-product.html.
> - Chime in the template parsing process to remove the description block for Product Catalog template.
> - Remove the description block.

I'll throw another option there, maybe unpopular 😬 what about just keeping the description block in there?

Reply:
> I'll throw another option there, maybe unpopular 😬 what about just keeping the description block in there?

@albarin Aha! It's a legit one. Let's consider it also.

Reply:
>I'll throw another option there, maybe unpopular 😬 what about just keeping the description block in there?

Definitely an option for now, the only problem is that it does not reflect what users will experience on the frontend. I wonder if its possible to put a note next to these blocks perhaps which explains "On the main Product Catalog page this information will not be visible" etc.

Just cc'ing @vivialice into this incase she has any ideas or would like to weigh in on one of the options.